CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape.uc

```diff
@@ -1126,8 +1126,35 @@ simulated function bool OnUnrealCommand(int cmd, int arg)
 
 simulated function OnRemoved()
 {
+	local XComGameStateHistory History;
+	local XComGameState NewGameState;
+	local XComGameState_CovertAction ActionState;
+	local bool bModified;
+
 	super.OnRemoved();
 
+	// Turn off the "Now Available" flag for any new covert actions
+	History = `XCOMHISTORY;
+	NewGameState = class'XComGameStateContext_ChangeContainer'.static.CreateChangeState("Turn off new Covert Action Now Available flag");
+	foreach arrActions(ActionState)
+	{
+		if( ActionState.bNewAction && ActionState.CanActionBeDisplayed())
+		{
+			ActionState = XComGameState_CovertAction(NewGameState.ModifyStateObject(class'XComGameState_CovertAction', ActionState.ObjectID));
+			ActionState.bNewAction = false;
+			bModified = true;
+		}
+	}
+
+	if( bModified )
```

Comment:
Please use the CHL code style: https://github.com/X2CommunityCore/X2WOTCCommunityHighlander#when-contributing-please